src/dfcx_scrapi/tools/search_util.py

```diff
@@ -30,6 +30,15 @@
 from google.cloud.dialogflowcx_v3beta1 import types
 from google.oauth2 import service_account
 
+from proto.marshal.collections.maps import MapComposite
+from proto.marshal.collections.repeated import RepeatedComposite
+
+# Type aliases
```

Comment:
@SeanScripts @skyyeat I don't think these global type aliases are necessary.

- DFCXFlow is not used anywhere in this class
- The other 3 are only used 1 time, so there's no real savings or reusability here